### PR TITLE
Adding Socketio-tester App v1.0,1

### PR DESCRIPTION
**Pull request by gobinathm**
- New Application Inclusion 

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/socket-io-tester.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'socket-io-tester' do
+  version '1.0.1'
+  sha256 '07cc306ba370a458966afd42bc287e556a40aa41a5d0c9aeab1719db0e648b9d'
+
+  # github.com/AppSaloon/socket.io-tester was verified as official when first introduced to the cask
+  url "https://github.com/AppSaloon/socket.io-tester/releases/download/v#{version}/socket-io-tester-darwin-x64.zip"
+  name 'socket-io-tester.app'
```
Comment (vitorgalvao) [inline Casks/socket-io-tester.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/socket-io-tester.rb (hunk 1 above)
Comment (gobinathm) [inline Casks/socket-io-tester.rb]: already check this & does not have an appcast

```
$ ../developer/bin/find_sparkle_appcast ~/Desktop/socket-io-tester-darwin-x64/socket-io-tester.app/
It appears this app does not have a Sparkle appcast
```

Comment (miccal) [inline Casks/socket-io-tester.rb]: If there is a Github releases page, it has an `appcast`. Please read the documentation @vitorgalvao linked to for instructions on how to find it.

Comment (gobinathm) [inline Casks/socket-io-tester.rb]: @vitorgalvao @miccal Sorry my mistake, i have added appcast url

**Pull request merged**